### ISSUE TITLE
Login works via UserID not UserName

### ISSUE THREAD

**Issue by DimitryRakhlei** [COMP4711-Asn01/StockTickerAgent#37]
I don't think users should have control over what UserID they get. From what I know it is just a number given to them by the system.

We can fix it later though.

**Comment by eddyL1111** [comment]
Yeah, I was doubting on how to do it, so I just added username as a way of only displaying the name that the user wants and not the ID.
